src/Nox.Ui.Blazor.Lib/Components/NoxTypes/EditEntityExactlyOne.razor

```diff
@@ -0,0 +1,24 @@
+﻿@namespace Nox.Ui.Blazor.Lib.Components.NoxTypes
+@using MudBlazor
+@using Nox.Ui.Blazor.Lib.Models
+
+@if (SelectionList != null && SelectionList.Count > 0)
+{
+    <MudSelect Label="@Title"
+               Variant="Variant.Outlined"
+               FullWidth="true"
+               Clearable="true"
+               Required="true"
```

Comment:
maybe we should add parameter if it's required or not, as it might not be required for zeroOrOne relationships

Reply:
within latest commit